ios/build.sh

```diff
@@ -1,13 +1,29 @@
 #!/usr/bin/env bash
 
-set -e
-# set -x
-# set -u <-- difficult to test if variable is set this way...
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
 
-./kmbuild.sh $*
+# This script runs from its own folder
+cd "$THIS_SCRIPT_PATH"
 
-if [ ! -z "$RELEASE_OEM" ]; then
-  pushd ../oem/firstvoices/ios > /dev/null
-  ./build.sh -lib-nobuild $*
-  popd > /dev/null
-fi
+# Include our resource functions; they're pretty useful!
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+# Please note that this build script (understandably) assumes that it is running on Mac OS X.
+verify_on_mac
+
+builder_describe "Builds Keyman Engine and the Keyman app for use on iOS devices - iPhone and iPad." \
+  "clean" \
+  "configure" \
+  "build" \
+  ":engine          Builds KeymanEngine.xcframework, usable by our main app and by third-party apps" \
```

Comment:
Would this go down to :engine=engine/KMEI...?

```suggestion
  ":engine=engine Builds KeymanEngine.xcframework, usable by our main app and by third-party apps" \
```

Reply:
As is, it just goes to `ios/engine/build.sh`.  (There is no `ios/engine/KMEI/build.sh`.)  I felt that `:engine=engine` was redundant.